MapboxNavigation/FeedbackViewController.swift

```diff
@@ -181,6 +189,7 @@ extension FeedbackViewController: UICollectionViewDataSource {
     
     func scrollViewDidScroll(_ scrollView: UIScrollView) {
         abortAutodismiss()
+        progressBar.progress = 1
```

Comment:
Minor enhancement, it might make more sense to move this into `dismissFeedback()`.

Reply:
@bsudekum so are you saying, let invoke `dismissFeedback()` here instead and move `line # 192` into the `dismssFeedback()` function, because it already calls `abortAutodismiss()` there. https://github.com/mapbox/mapbox-navigation-ios/pull/1436/files#diff-a25e8b3bfe9ceec368b83a004a6d1708R129

Reply:
I guess I'm saying move it to where the view is cleaned up.

Reply:
@bsudekum yeah, I concur. A slight concern about intrusive changes. But, @frederoni I'm curious to know if you could suggest a change which will still maintain the current stability of our SDK. Otherwise, this should actually suffice without any side-effects.

Reply:
As long as the collection view doesn't bounce and the scroll view’s (or collection view in this case) content size matches the bounds, so that isn't scrollable. Also, add a note why we do this b/c it's not an obvious workaround.